client/src/elm/Pages/GlobalCaseManagement/View.elm

```diff
@@ -939,24 +934,127 @@ viewPrenatalLabsEntry :
     -> PrenatalLabsEntryData
     -> Html Msg
 viewPrenatalLabsEntry language data =
+    viewLabsEntry language
+        data.personName
+        data.state
+        data.label
+        (PrenatalRecurrentEncounterPage data.encounterId)
+
+
+viewLabsEntry :
+    Language
+    -> String
+    -> LabsEntryState
+    -> String
+    -> UserPage
+    -> Html Msg
+viewLabsEntry language personName state label targetPage =
     let
         entryStateClass =
             "due "
-                ++ (case data.state of
-                        PrenatalLabsEntryClosingSoon ->
+                ++ (case state of
+                        LabsEntryClosingSoon ->
                             "overdue"
 
-                        PrenatalLabsEntryPending ->
+                        LabsEntryPending ->
                             "this-week"
                    )
     in
     div [ class "follow-up-entry" ]
-        [ div [ class "name" ] [ text data.personName ]
-        , div [ class entryStateClass ] [ translateText language <| Translate.PrenatalLabsEntryState data.state ]
-        , div [ class "assesment center" ] [ text data.label ]
+        [ div [ class "name" ] [ text personName ]
+        , div [ class entryStateClass ] [ translateText language <| Translate.LabsEntryState state ]
+        , div [ class "assesment center" ] [ text label ]
         , div
             [ class "icon-forward"
-            , onClick <| SetActivePage <| UserPage <| PrenatalRecurrentEncounterPage data.encounterId
+            , onClick <| SetActivePage <| UserPage targetPage
             ]
             []
         ]
+
+
+viewNCDLabsPane :
+    Language
+    -> NominalDate
+    -> Dict NCDLabsResultsId NCDLabsResults
+    -> ModelIndexedDb
+    -> Model
+    -> Html Msg
+viewNCDLabsPane language currentDate itemsDict db model =
+    let
+        filteredItemsDict =
+            Dict.filter
+                (\_ item ->
+                    -- We know that item is not resolved, if resolution
+                    -- date is a future date.
+                    Date.compare currentDate item.value.resolutionDate == LT
+                )
+                itemsDict
+
+        entries =
+            generateNCDLabsEntries language currentDate filteredItemsDict db
+
+        content =
+            if List.isEmpty entries then
+                [ translateText language Translate.NoMatchesFound ]
+
+            else
+                List.map (viewNCDLabsEntry language) entries
+    in
+    div [ class "pane" ]
```

Comment:
```suggestion
    div [ class "pane ncd-labs" ]
```

Reply:
Not required for styling, and other types ofpanes do add extra class, so I'd rather leave it as is.